Gordon360/Models/AA_ApartmentApplications.cs

```diff
@@ -15,9 +15,8 @@ namespace Gordon360.Models
     public partial class AA_ApartmentApplications
```

Comment:
This was actually related to rebuilding the .edmx files. I did a less global refresh this time, since I had only changed a couple of columns on tables rather than adding/deleting tables. That refresh still picked up changes to these auto-generated Models as well though.

These are currently not referenced anywhere in our code though, they are simply artifacts in the database, so it shouldn't cause any issues to include them.